test/tests/fips_test_suite/services.rs

```diff
@@ -140,6 +298,103 @@ pub fn exec_cmd_stash_measurement<T: HwModel>(hw: &mut T) {
     assert_eq!(stash_measurement_resp.dpe_result, 0);
 }
 
+pub fn exec_fw_info<T: HwModel>(hw: &mut T) {
+    let payload = MailboxReqHeader {
+        chksum: caliptra_common::checksum::calc_checksum(u32::from(CommandId::FW_INFO), &[]),
+    };
+    let fw_info_resp = mbx_send_and_check_resp_hdr::<_, FwInfoResp>(
+        hw,
+        u32::from(CommandId::FW_INFO),
+        payload.as_bytes(),
+    )
+    .unwrap();
+
+    // Verify command-specific response data
+    assert_eq!(fw_info_resp.fmc_revision, image_revision().unwrap());
```

Comment:
This is related to what I haven't documented yet, but we'll need to be able to cover older versions of our collateral as well with the test suite. So there are structs for the expected data (like ROM_EXP_1_0_1 and ROM_EXP_CURRENT). So ideally we'd put this image_revision() call in the *_EXP_CURRENT structs. However, we may need to move populating the *_EXP_CURRENT structs to runtime because we can't call a function to init a const. Or maybe there is a cleaner solution in rust?

The other catch with this particular case is specifying a single commit ID for a released version. Since we package all our collateral into a release, you'd have to pull different collateral from different releases for the commit IDs to match. But we can worry about that detail later. Maybe we can expect the user to populate their own struct when it comes to Si where this stuff gets harder.